### PR TITLE
Feature/channels env

### PR DESCRIPTION
**Pull request by vanderian**

### CODE REVIEW

File: launch/README.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+## Description
+Provides an easy way to set up a local chain based on .yml configs.
+For the Mangata nodes we are currently using the dev Docker image, which needs to be built locally.
+
+## Run
+build the mangata node & create image
```
Comment (mateuszaaa) [inline launch/README.md:6]:
```suggestion
## Run
 Install `parachain-launch` globally

> yarn global add @open-web3/parachain-launch

Build the mangata node & create image
```

**Review by mateuszaaa — CHANGES_REQUESTED** [review]
minor changes

File: launch/config.yml [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+  flags:
+    - --rpc-methods=unsafe
+    - --wasm-execution=compiled
+    - --execution=wasm
+  nodes:
+    - name: alice
+    - name: bob
+
+parachains:
+- image: mangatasolutions/mangata-node:dev
```
Comment (mateuszaaa) [inline launch/config.yml:20]:
```suggestion
- image: mangatasolutions/mangata-node:dev
  env:
    RUST_LOG: error
```
to disable logging at the time the chainspecs are generated

File: launch/README.md (hunk 1 above)
Comment (vanderian) [reply launch/README.md:6]: we are not using the '@open-web3/parachain-launch' i had to fork and make some changes

**Review by mateuszaaa — APPROVED** [review]
lgtm

**Pull request merged**